python-sdk/src/astro/settings.py

```diff
@@ -41,3 +41,8 @@
 AUTO_ADD_INLETS_OUTLETS = conf.getboolean(
     "astro_sdk", "auto_add_inlets_outlets", fallback=True
 )
+
+
+COLUMN_CAPITALIZATION = conf.get(
+    "astro_sdk", "column_capitalization", fallback="original"
+)
```

Comment:
We should add docs about it in https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/configurations.rst

Reply:
Also, you have the following in the PR description

>Does this introduce a breaking change?
> Nope

But this does introduce a breaking change. When writing the release notes, we should ask users to change the config if needed if they relied on this behavior and mention that we consider it a "bug" hence the change

Reply:
Why is that a breaking change? The default does not change as we are setting `COLUMN_CAPITALIZATION` default/fallback to `original`.

Reply:
+1

Reply:
Thanks @kaxil for pointing this out 👍